docs/pre-consumption/decrypt-pdf.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Decrypt PDFs
+---
+
+# Decrypt PDFs before consumption
+
+Encrypted PDFs are one way to rain on your parade with paperless-ngx - even if you know the password. You get no thumbnail, no OCR, you are constantly nagged for the password when you want to view the file. So let's get rid of the encryption!
+
+## Setup
+
+_decrypt-pdf_ consists of
+
+1. the script itself
+2. the password files (insecure.pwd.txt and personal.pwd.txt)
+   * **personal.pwd.txt** is slated for your personal passwords, it's also added to the local _.gitignore_ file
+   * **insecure.pwd.txt** contains some of the most prolific passwords I could find
+   * you can create other files in the same manner to try passwords from dumps, "most common password in XXX" lists etc.
```

Comment:
Currently, it renders like this, enumerating the details on the password files:

<img width="796" alt="grafik" src="https://github.com/user-attachments/assets/4390a8fa-6138-49df-8372-bf3e593333ee">


But isn't it meant to render like this, with the details further indented? If so, please add a space before each unnumbered list item.

<img width="776" alt="grafik" src="https://github.com/user-attachments/assets/189e5168-fe89-4df7-8580-2e6dfe964958">

Reply:
I've checked out your code and ran `mkdocs serve`